app/assets/javascripts/modules/coronavirus-local-restrictions.js

```diff
@@ -0,0 +1,37 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function CoronavirusLocalRestrictions () {}
+
+  CoronavirusLocalRestrictions.prototype.start = function (element) {
+    this.addPostcodeFormSubmitListener(element[0])
+    this.addErrorMessageEvent(element[0])
+  }
+
+  CoronavirusLocalRestrictions.prototype.addPostcodeFormSubmitListener = function (element) {
+    element.querySelector('.coronavirus-local-restrictions__postcode-form')
+      .addEventListener('submit', function(event){
+        var options = {
+          transport: "beacon"
+        }
+
+        GOVUK.analytics.trackEvent("postcodeSearch:local_lockdown", "postcodeSearchStarted", options)
+      })
+  }
+
+  CoronavirusLocalRestrictions.prototype.addErrorMessageEvent = function (element) {
+    var errorMessage = element.querySelector('.govuk-error-summary__body')
```

Comment:
> Forgive me, looking at this for the first time with very little context - jQuery is loaded by we're using ES6?


We're moving away from jQuery and therefore anything new we write, we try to avoid adding new jQuery in.

Reply:
> I'm also wondering if data- vs class is a better hook - eg what if the classes change?

Yes you have a point, I will look into whether this is possible with the component as it is

Reply:
Changed.